tests/kernel/msgq/msgq_api/src/test_msgq_contexts.c

```diff
@@ -429,8 +432,9 @@ void test_msgq_full(void)
 	ret = k_sem_init(&end_sema, 0, 1);
 	zassert_equal(ret, 0, NULL);
 
-
 	k_msgq_put(&msgq1, &data[0], K_NO_WAIT);
```

Comment:
need to capture ret here?

Reply:
Oh, so sorry, miss it, thank so much for your correction.